Automattic/jetpack
# RNA: Connection Screen

#### Changes proposed in this Pull Request:
- add `ConnectScreen` component implements the connection screen
- add `ImageSlider` component for the image sliding/rotation on the right side of the connection screen
- integration of the new `ConnectScreen` component into the Connection Manager UI
- moving the `getRedirectUrl()` function into the RNA Components package

#### Jetpack product discussion
1200295095943283-as-1200465631534665
p1HpG7-caw-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
_If you're using a JN site, you need to install [this plugin](https://gist.github.com/sergeymitr/bc005e1b893e778c03e4f43debb91c30) to activate the Connection Manager page._

1. Disconnect Jetpack.
2. Go to "Tools -> Connection Manager".
3. Confirm that the page looks like this:
<img width="1563" alt="Screen Shot 2021-07-13 at 4 38 16 PM" src="https://user-images.githubusercontent.com/1341249/125461699-9d1c185f-f4f7-480d-b0eb-5a5f45e0261e.png">
4. Click "Set up Jetpack". The button should become inactive, then you should be redirected to Calypso connection flow.
5. Approve the connection in Calypso. Confirm that you're redirected back to the Connection Manager, and the page says "Site and User Connected".
